pages/linux/ncdu.md

```diff
@@ -1,11 +1,16 @@
 # ncdu
 
 > Disk usage analyzer with an ncurses interface.
+> More information: <https://manned.org/ncdu>.
 
 - Analyze the current working directory:
 
 `ncdu`
 
+- Colorize output:
+
+`ncdu --color dark`
```

Comment:
```suggestion
- Colorize output (available colors `off`, `dark`):

`ncdu --color {{dark}}`
```

Reply:
Since `dark` is the only available colorising option for now I thought might as well omit it. This is fine too.